draft-dr-dprive-tls-and-dtls-authentication.txt

```diff
@@ -141,91 +149,86 @@ Internet-Draft            (D)TLS Authentication            November 2015
-      include:
 
 
 
-Dickinson & Reddy         Expires May 19, 2016                  [Page 3]
+Dickinson & Reddy         Expires May 21, 2016                  [Page 3]
 
 Internet-Draft            (D)TLS Authentication            November 2015
 
 
       *  X.509 certificate
 
-      *  DNSSEC chain to TLSA record
+      *  DNSSEC validated chain to a TLSA record
 
    o  SPKI Pinsets: [I-D.ietf-dprive-dns-over-tls] describes the use of
       cryptographic digests to "pin" public key information in a manner
       similar to HPKP [RFC7469].  An SPKI pinset is a collection of
-      these pins that constrains a DNS Resolver.
+      these pins that constrains a DNS server.
+
+   o  Reference Identifier: a Reference Identifier as described in
+      [RFC6125], constructed by the DNS client when performing TLS
+      authentication of a DNS server.
 
    o  (D)TLS: Transport Layer Security [RFC5246] and (possibly) Datagram
       Transport Layer Security [RFC6347]
 
-   o  Privacy-protective DNS Server: A DNS Resolver that:
+   o  Privacy-enabling DNS server: A DNS server that:
 
-      *  Accepts DNS-over-TLS [I-D.ietf-dprive-dns-over-tls] connections
-         and MAY accept DNS-over-DTLS [I-D.ietf-dprive-dnsodtls]
-         connections.
+      *  MUST implement DNS-over-TLS [I-D.ietf-dprive-dns-over-tls] and
+         MAY implement DNS-over-DTLS [I-D.ietf-dprive-dnsodtls].
 
       *  Can offer at least one of the credentials described in
          Section 9.
 
-      *  Offers the (D)TLS profile described in Section 12.
+      *  Implements the (D)TLS profile described in Section 12.
 
-3.  Scope
+      [QUESTION: This assumes that all servers that implement DNS-over-
+      DTLS will be required to implement DNS-over-TLS to support
+      truncation.  This is implied in the msg-flows draft but isn't yet
+      stated in the current DNS-over-DTLS draft.  But I believe it will
+      be in the next version? ]
 
-   This document is limited to strong, domain-name-based authentication
-   of recursive resolvers by DNS clients (stubs and forwarders), and the
-   TLS profiles needed to support this.  As such, the following things
-   are out of scope:
+3.  Scope
 
-   o  Authentication of authoritative nameservers by recursive
-      resolvers.
+   This document is limited to domain-name-based authentication of DNS
+   servers by DNS clients (as defined in the terminology section), and
+   the TLS profiles needed to support this.  As such, the following
+   things are out of scope:
 
-   o  Authentication of DNS clients by servers.
+   o  Authentication of authoritative servers by recursive resolvers.
 
-   o  DNS clients (stubs and resolvers) using TLS in an "opportunistic"
-      mode [RFC7435].  These clients should be willing to accept failed
-      authentication, which is strictly preferable to cleartext.
-      However, this document does briefly describe an extension to
-      opportunistic connections in Section 6.
```

Comment:
As I said, I don't think we can say this is out of scope if we are effectively adding new requirements relating to DTLS using this, etc.